src/content/config.ts

```diff
@@ -2,7 +2,7 @@ import { defineCollection, z } from 'astro:content';
 
 const guides = defineCollection({
     schema: z.object({
-        title: z.string(),
+        title: z.string().optional(),
```

Comment:
Do we prefer to make this optional, or should we just make "Set up Supabase database" as the title for the page?

Reply:
Yep, lets use "Set up Supabase database" for the title

Reply:
sounds good - I think we can undo this change then 👍